tensorflow/lite/micro/python/interpreter/src/python_ops_resolver.cc

```diff
@@ -91,6 +91,12 @@ PythonOpsResolver::PythonOpsResolver() {
   AddReshape();
   AddResizeBilinear();
   AddResizeNearestNeighbor();
+  // TODO() Maybe for the python version we only need Rfft since that will
+  // register all and at runtime we change based on type?
+  AddRfft();
+  AddRfftFloat();
+  AddRfftInt16();
+  AddRfftInt32();
```

Comment:
we don't need these variants from Python since binary size is not a concern at all.